### DISCUSSION TITLE
Hello all Can you help me return the money? I sent Ethereum to buy Manta but I have not received the coins for a month yet      0x144a14231a9c64800e54bf4ec2f16cbbeac863d1

### DISCUSSION THREAD

**Discussion by MOIUJIBKHALIL** [blockscout/blockscout#9431] [General]
Hello all
Can you help me return the money?
I sent Ethereum to buy Manta but I have not received the coins for a month yet
![photo_2024-02-20_16-11-14](https://github.com/blockscout/blockscout/assets/159659955/c2969c3c-de3a-4d2e-9002-ab2daee888a2)
![photo_2024-02-20_16-10-52](https://github.com/blockscout/blockscout/assets/159659955/5338597a-83f2-4e1a-97a1-ccd91e225970)
![photo_2024-02-20_16-10-58](https://github.com/blockscout/blockscout/assets/159659955/5110990f-5a98-4891-b414-25a1b8871fd1)
![photo_2024-02-20_16-11-01](https://github.com/blockscout/blockscout/assets/159659955/9f9c037f-6646-4d08-8f75-67131fc6e94e)
![photo_2024-02-20_16-11-04](https://github.com/blockscout/blockscout/assets/159659955/4e08603b-b01f-4367-a772-a3231ff11dab)
![photo_2024-02-20_16-11-11](https://github.com/blockscout/blockscout/assets/159659955/91046486-d34a-4774-be7d-f209dfda258a)

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.
